test/functional/helpers.lua

```diff
@@ -747,41 +739,14 @@ local function alter_slashes(obj)
   end
 end
 
-local function compute_load_factor()
-  local timeout = 200
-  local times = {}
-
-  clear()
-
-  for _ = 1, 5 do
-    source([[
-      let g:val = 0
-      call timer_start(200, {-> nvim_set_var('val', 1)})
-      let start = reltime()
-      while 1
-        sleep 10m
-        if g:val == 1
-          let g:waited_in_ms = float2nr(reltimefloat(reltime(start)) * 1000)
-          break
-        endif
-      endwhile
-    ]])
-    table.insert(times, nvim_eval('g:waited_in_ms'))
-  end
-
-  session:close()
-  session = nil
-
-  local longest = math.max(unpack(times))
-  local factor = (longest + 50.0) / timeout
-
-  return factor
-end
-
--- Compute load factor only once.
-local load_factor = compute_load_factor()
```

Comment:
@mhinz this was running for all tests, even those that don't need `load_adjust()`.  I guess that could slow down the tests.